src/core-components/ConnectWallet/ConnectWallet.stories.tsx

```diff
@@ -31,7 +28,7 @@ export const Template: Story = () => (
     <Row gutter={[{ xs: 8, sm: 16, md: 24 }, 8]}>
       <Col span={8}>
         <ConnectWallet
-          type="default"
+          type="connected"
           balance="0 ERG"
```

Comment:
Let's use balance and currency separately, coz balance may have bigint type in the future. Now leave it as number